man/systemd.mount.xml

```diff
@@ -148,16 +148,12 @@
   <refsect1>
     <title><filename>fstab</filename></title>
 
-    <para>Mount units may either be configured via unit files, or via
-    <filename>/etc/fstab</filename> (see
-    <citerefentry project='man-pages'><refentrytitle>fstab</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-    for details). Mounts listed in <filename>/etc/fstab</filename>
-    will be converted into native units dynamically at boot and when
-    the configuration of the system manager is reloaded. In general,
-    configuring mount points through <filename>/etc/fstab</filename>
-    is the preferred approach. See
```

Comment:
I'm reading https://github.com/systemd/systemd/pull/3152#issuecomment-216072070
And I think we shouldn't drop comment about /etc/fstab being preferrable over .mount units